Gemfile

```
@@ -4,4 +4,6 @@ ruby '2.6.0'
 gem 'airrecord'
 gem 'sinatra'
 gem 'dotenv'
-gem 'activesupport'
\ No newline at end of file
+gem 'rspec'
```

this should be like this (to prevent test library's being installed in prod)
```suggestion
group :test do
  gem 'rspec'
end
```

---

Ah thanks I did not know about this syntax in ruby!